rskj-core/src/main/java/co/rsk/peg/whitelist/WhitelistResponseCode.java

```diff
@@ -0,0 +1,19 @@
+package co.rsk.peg.whitelist;
+
+public enum WhitelistResponseCode {
+    GENERIC_ERROR(-10),
+    INVALID_ADDRESS_FORMAT_ERROR(-2),
```

Comment:
```suggestion
    INVALID_ADDRESS_FORMAT(-2),
```
I think the name already implies it's an error

Reply:
I agree, but we need to decide on a new name for "GENERIC" since it will look weird with just `GENERIC`, at least to me. Wdyt?

Reply:
I'd keep GENERIC_ERROR, and UNKOWN_ERROR. My proposal is just for INVALID_ADDRESS_FORMAT

Reply:
Done!